tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1225,8 +1227,47 @@ Status Converter::BuildCudaEngine(
   }
 
 #if IS_TRT_VERSION_GE(8, 0, 0, 0)
-  builder_config->setFlag(nvinfer1::BuilderFlag::kSPARSE_WEIGHTS);
-  VLOG(1) << "Setting sparsity for TensorRT8!";
+  enum class SparseComputeMode {
+    DISABLED,
+    ENABLED,
+    BENCHMARK
+  };
+
+  static SparseComputeMode sparse_compute_mode = [](bool _enable_sparse_compute)
+  {
+    SparseComputeMode _sparse_compute_mode;
+    int64 _sparse_benchmark_mode;
+    /*This feature is only to be used for benchmarking purpose. This feature
+    artificially introduce a sparse weight pattern compatible with Sparse
+    TensorCores of NVIDIA Ampere GPU architecture. It will complete corrupt
+    the numerical values of the computation, it shall only be used to evaluate
+    the benefit of using sparse computation for inference*/
```

Comment:
s/, it shall/ and shall/
Please add a period at the end of the comment.

Reply:
Fixed